api/layer/neofs_mock.go

```diff
@@ -270,19 +270,13 @@ func (t *TestNeoFS) CreateObject(_ context.Context, prm PrmObjectCreate) (oid.ID
 	obj.SetOwnerID(&prm.Creator)
 	t.currentEpoch++
 
-	if prm.Multipart != nil && prm.Multipart.SplitID != "" {
-		var split object.SplitID
-		if err := split.Parse(prm.Multipart.SplitID); err != nil {
-			return oid.ID{}, fmt.Errorf("split parse: %w", err)
-		}
-		obj.SetSplitID(&split)
-
+	if prm.Multipart != nil {
```

Comment:
so the whole `if` works right only because we pray it is called right? i mean a link object should never have a previous one sert, a regular object's payload should never be rewritten, etc